testdata/p4_16_samples_outputs/psa-table-hit.p4.spec

```diff
@@ -0,0 +1,76 @@
+
+struct EMPTY_M {
+	bit<32> psa_ingress_parser_input_metadata_ingress_port
+	bit<32> psa_ingress_parser_input_metadata_packet_path
+	bit<32> psa_egress_parser_input_metadata_egress_port
+	bit<32> psa_egress_parser_input_metadata_packet_path
+	bit<32> psa_ingress_input_metadata_ingress_port
+	bit<32> psa_ingress_input_metadata_packet_path
+	bit<64> psa_ingress_input_metadata_ingress_timestamp
+	bit<8> psa_ingress_input_metadata_parser_error
+	bit<8> psa_ingress_output_metadata_class_of_service
+	bit<8> psa_ingress_output_metadata_clone
+	bit<16> psa_ingress_output_metadata_clone_session_id
+	bit<8> psa_ingress_output_metadata_drop
+	bit<8> psa_ingress_output_metadata_resubmit
+	bit<32> psa_ingress_output_metadata_multicast_group
+	bit<32> psa_ingress_output_metadata_egress_port
+	bit<8> psa_egress_input_metadata_class_of_service
+	bit<32> psa_egress_input_metadata_egress_port
+	bit<32> psa_egress_input_metadata_packet_path
+	bit<16> psa_egress_input_metadata_instance
+	bit<64> psa_egress_input_metadata_egress_timestamp
+	bit<8> psa_egress_input_metadata_parser_error
+	bit<32> psa_egress_deparser_input_metadata_egress_port
+	bit<8> psa_egress_output_metadata_clone
+	bit<16> psa_egress_output_metadata_clone_session_id
+	bit<8> psa_egress_output_metadata_drop
+}
+metadata instanceof EMPTY_M
+
+action NoAction_0 args none {
+	return
+}
+
+action remove_header args none {
+	invalidate h
+	return
+}
+
+action ifHit args none {
+	invalidate h
+	return
+}
+
+table tbl_0 {
+	key {
+		h.srcAddr exact
+	}
+	actions {
+		NoAction_0
+		remove_header
+	}
+	default_action NoAction_0 args none 
+	size 0
+}
+
+
+table tbl_ifHit {
+	actions {
+		ifHit
+	}
+	default_action ifHit args none 
+	size 0
+}
+
+
+apply {
+	rx m.psa_ingress_input_metadata_ingress_port
+	extract h
+	table tbl_0
+	jmph LABEL_0END
```

Comment:
is this the correct output or is it inverted?

Reply:
This is correct to my understanding, see comment from Crisitian below.

>“Jump if table lookup hit” (jmph) example:
table vxlan_table
//some instructions here, but none of them is “table”, as jmph and jmpnh will refer to the table from the last “table” instruction
jmph GO_HERE_ON_VXLAN_TABLE_HIT
//some instructions
GO_HERE_ON_VXLAN_TABLE_HIT:
//some instructions to execute on vxlan_table hit

Reply:
This still seems to me to be inverted. On hit you skip the tbl_ifHit application.

Reply:
Thanks, I fixed the issue.